manifests/init.pp

```diff
@@ -39,16 +62,15 @@
 
   if $control_socket == true {
     warning(@(WARN/L)
-      Use of the control_socket parameter in the freeradius class is deprecated. \
-      Please use the freeradius::control_socket class instead.
-      |-WARN
+        Use of the control_socket parameter in the freeradius class is deprecated. \
+        Please use the freeradius::control_socket class instead.
+        |-WARN
     )
   }
 
   # Always restart the service after every module operation
-  Freeradius::Module {
-    notify => Service[$freeradius::fr_service]
-  }
+  Freeradius::Module <||>
+  ~> Service[$freeradius::fr_service]
```

Comment:
Hope that change is fine with you guys. Just wanted to get ride of that Default definition `Freeradius::Module {}` since on never puppet version it is best practice to do so.
Is now affects really all definded `Freeradius::Module` before only the following `Freeradius::Module` resources in the scope.

Reply:
Mhm in the end these lines could also be removed since the file resources inside `freeradius::module` already define the needed restart of the Service:
https://github.com/djjudas21/puppet-freeradius/blob/4f118df4aeafe5cdc87a1f2deb006ee86253f24d/manifests/module.pp#L24-L42